examples/svm/query/query.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	_ "embed"
+
+	chaintypes "github.com/FluxNFTLabs/sdk-go/chain/types"
+	chainclient "github.com/FluxNFTLabs/sdk-go/client/chain"
+	"github.com/FluxNFTLabs/sdk-go/client/common"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
+	"github.com/gagliardetto/solana-go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var (
+	privKey []byte
+)
+
+func main() {
+	network := common.LoadNetwork("local", "")
+	kr, err := keyring.New(
+		"fluxd",
+		"file",
+		os.Getenv("HOME")+"/.fluxd",
+		strings.NewReader("12345678"),
+		chainclient.GetCryptoCodec(),
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	// init grpc connection
+	cc, err := grpc.Dial(network.ChainGrpcEndpoint, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		panic(err)
+	}
+
+	// init client ctx
+	clientCtx, _, err := chaintypes.NewClientContext(
+		network.ChainId,
+		"user1",
+		kr,
+	)
+	if err != nil {
+		panic(err)
+	}
+	clientCtx = clientCtx.WithGRPCClient(cc)
+
+	dir, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+
+	privKey, err = os.ReadFile(dir + "/examples/svm/build/counter-contract-keypair.json")
+	if err != nil {
+		panic(err)
+	}
+
+	var privKeyBz []byte
+	if err := json.Unmarshal(privKey, &privKeyBz); err != nil {
+		panic(err)
+	}
+	privKey := &ed25519.PrivKey{Key: privKeyBz}
+	programId := solana.PublicKeyFromBytes(privKey.PubKey().Bytes())
+
+	// init chain client
+	chainClient, err := chainclient.NewChainClient(
+		clientCtx,
+		common.OptionGasPrices("500000000lux"),
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	counterPubkey, _, err := solana.FindProgramAddress([][]byte{
+		[]byte("counter"),
+	}, programId)
+	if err != nil {
+		panic(err)
+	}
+
+	accountInfo, err := chainClient.GetSvmAccount(context.Background(), counterPubkey.String())
+	if err != nil {
+		log.Fatalf("Failed to fetch PDA account: %v", err)
+	}
+
+	if len(accountInfo.Account.Data) >= 16 {
+		counterValue := uint64(0)
+		for i := 0; i < 8; i++ {
+			counterValue |= uint64(accountInfo.Account.Data[8+i]) << (i * 8)
+		}
+		fmt.Printf("Counter value: %d\n", counterValue)
+	}
```

Comment:
Just use golang binary lib, you don't have to parse it yourself, first 8 bytes is the account discriminator, ruled by anchor

`https://pkg.go.dev/encoding/binary`

Tip: Do some research for standard stuff, the library must be out there somewhere